packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte

```diff
@@ -182,6 +182,15 @@
         indexes,
       })
       dispatch("updatecolumns")
+      if (
+        saveColumn.type === LINK_TYPE &&
+        saveColumn.relationshipType === RelationshipTypes.MANY_TO_MANY
+      ) {
+        // Fetching the new tables
+        tables.fetch()
+        // Fetching the new relationships
+        datasources.fetch()
```

Comment:
I wish there was a cleaner way to just update state with the new data and avoid hitting the API again, but in this case because it affects both the datasource docs and table docs (and multiple tables at that) I'm happy to just refresh everything.

Reply:
We could theoretically update the table API to return all the affected entities when a relationship is added (or anything that affects multiple tables) but it might be a fair bit of extra work.